async-nats/src/lib.rs

```diff
@@ -11,41 +11,50 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! A Rust async bleeding edge client for the NATS.io ecosystem.
+//! A Rust asynchronous client for the NATS.io ecosystem.
 //!
-//! `git clone https://github.com/nats-io/nats.rs`
+//! To access the repository, you can clone it by running:
 //!
-//! NATS.io is a simple, secure and high performance open source messaging
-//! system for cloud native applications, `IoT` messaging, and microservices
+//! ```bash
+//! git clone https://github.com/nats-io/nats.rs
+//! ````
+//! NATS.io is a simple, secure, and high-performance open-source messaging
+//! system designed for cloud-native applications, IoT messaging, and microservices
 //! architectures.
 //!
-//! For sync API refer <https://crates.io/crates/nats>
+//! For the synchronous API, you can refer to:
```

Comment:
Mention here that sync api is deprecated (it is not yet, but will be very soon, and no one looking at async repo should have ideas to go there.